.github/workflows/support-stackoverflow.yml

```diff
@@ -25,7 +25,7 @@ jobs:
             We use GitHub issues as a bug and feature requests tracker, however,
             this issue appears to be a support request.
 
-            For support, please check out https://mui.com/components/data-grid/getting-started/#support. Thanks!
+            For support, please check out https://mui.com/getting-started/support/. Thanks!
```

Comment:
To be consistent with [the issue templates](https://github.com/mui/mui-x/blob/master/.github/ISSUE_TEMPLATE/3.pro-support.yml) I put the link of the core doc

I don't know if we should strengthen our Support section (or do an `Advanced Components - Support` page) and use it on the Issue templates AND on the Stackoverflow bot.